test/helpers.js

```diff
@@ -1,30 +0,0 @@
-// Helper function to determine whether this browser is supported by 
-// impress.js or not. Copied from impress.js itself.
-var _impressSupported = function() {
```

Comment:
The tests that checks when impress is not supported was removed for the reasons stated [here](https://github.com/impress/impress.js/pull/510/files#r55663313).